packages/components/src/components/post-tabs/post-tabs.scss

```diff
@@ -3,3 +3,22 @@
 :host {
   display: block;
 }
+
+:host([fullwidth='true']) .tabs-wrapper {
+  // Extra space outside the container
+
+  $extra-space: calc((100vw - var(--post-core-dimension-1280)) / 2);
+  $space-to-compensate: max(0px, $extra-space);
+  $total-offset-per-side: calc(
+    max(var(--post-core-dimension-40), 0.5 * var(--post-grid-gutter-x)) + $space-to-compensate
+  );
```

Comment:
Core token should never be used to style a component directly. Please check [how the containers are build](packages/styles/src/layout/_containers.scss) and use the same tokens to create the style for the full-width tabs.

Also, the tabs aren't currently full-width on tablet, can you correct that?